app/file.go

```diff
@@ -596,7 +596,7 @@ func (a *App) UploadFileX(channelId, name string, input io.Reader,
 	// Concurrently upload and update DB, and post-process the image.
 	wg := sync.WaitGroup{}
 
-	if !t.Raw && t.fileinfo.IsImage() {
+	if !t.Raw && t.fileinfo.IsImage() && t.fileinfo.MimeType != "image/svg+xml" {
```

Comment:
nit: Can we let `postprocessImage` decide that this isn't an image that it needs to handle? I realize that adds the overhead of a goroutine that will do effectively nothing, but it seems better to contain the business logic to that function instead.